tests/components/google/conftest.py

```diff
@@ -29,17 +49,66 @@
     "kind": "calendar#calendarListEntry",
     "backgroundColor": "#16a765",
     "description": "Test Calendar",
-    "summary": "We are, we are, a... Test Calendar",
+    "summary": TEST_API_ENTITY_NAME,
     "colorId": "8",
     "defaultReminders": [],
-    "track": True,
```

Comment:
This behavior was a little odd before as test_calendar.py was taking an object that looks like an API response, but using it in the context of a google_calendar.yaml file. Now, we totally separate how  google_calendary.yaml and the API objects are set up and it is consistent between tests.